### ISSUE TITLE
CacheAdapter cannot be used in the configuration file

### ISSUE THREAD

**Issue by beiguancyc** [parse-community/parse-server#8610]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

I prefer to configure Parse Server using a configuration file. In Parse Server 5.x, I used the following code to configure the cacheAdapter, but now in 6.x, I keep getting the error "The client is closed at Commander._RedisClient_sendCommand."
`
"cacheAdapter":{
    "module":"C:/Users/Administrator/AppData/Roaming/npm/node_modules/parse-server/lib/Adapters/Cache/RedisCacheAdapter",
    "options":{
	  "url":"redis://127.0.0.1:6379/1"
	}
  }
`

**Comment by mtrezza** [comment]
Please provide the exact Parse Server version you are using; and possibly the earliest Parse Server version where you started to encounter the issue if you already did that investigation.

**Comment by beiguancyc** [comment]
> Please provide the exact Parse Server version you are using; and possibly the earliest Parse Server version where you started to encounter the issue if you already did that investigation.

I have tried every official release of Parse Server before version 5.5.0, and it worked without any issues. However, starting from version 6.1.0 and onwards, I encounter an error. The error message is "The client is closed at Commander._RedisClient_sendCommand."
And this is my config file.
```
{
  "appId" : "aaa",
  "masterKey" : "aaa",
  "dotNetKey" : "aaa",
  "clientKey" : "aaa",
  "restAPIKey" : "aaa",
  "javascriptKey" : "aaa",
  "databaseURI" : "mongodb://aaa",
  "publicServerURL" : "https://aaa",
  "port" : "1111",
  "mountPath" : "/",
  "cloud" : "c:/aaa/cloud/main.js",
  "jsonLogs" : true,
  "mountGraphQL" : true,
  "logsFolder" : "c:/aaa/log/logsParse",
  "maxLogFiles" : "20",
  "maxUploadSize" : "2000mb",
  "allowClientClassCreation":false,
  "cacheAdapter":{
    "module":"C:/Users/Administrator/AppData/Roaming/npm/node_modules/parse-server/lib/Adapters/Cache/RedisCacheAdapter",
    "options":{
	  "url":"redis://127.0.0.1:6379/1"
	}
  }
}
```

**Comment by dblythy** [comment]
How are you starting Parse Server? `cacheAdapter?.connect` is called in `api.start()` (a new method in V6)

**Comment by beiguancyc** [comment]
> How are you starting Parse Server? `cacheAdapter?.connect` is called in `api.start()` (a new method in V6)

I saved the above configuration as config.json file and then executed the command `parse-server config.json` in the cmd. It works fine in version 5.x, but in version 6.x, it throws an error in the logs.
<img width="1129" alt="image" src="https://github.com/parse-community/parse-server/assets/12932389/c04efc9b-fc08-4373-8c45-049b4fd4cf41">

**Comment by dblythy** [comment]
Confirming I have replicated the issue and waiting to hear whether #8633 resolves the issue

**Comment by beiguancyc** [comment]
> Confirming I have replicated the issue and waiting to hear whether #8633 resolves the issue

After several days of testing, no errors have occurred again.

**Comment by mtrezza** [comment]
Closing, as this is a bug in Parse Server 6, which is in LTS support.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-alpha.27](https://github.com/parse-community/parse-server/releases/tag/7.0.0-alpha.27)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/7.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0](https://github.com/parse-community/parse-server/releases/tag/7.0.0)
